airflow/contrib/hooks/gcp_sql_hook.py

```diff
@@ -144,6 +144,96 @@ def delete_instance(self, project_id, instance):
         operation_name = response["name"]
         return self._wait_for_operation_to_complete(project_id, operation_name)
 
+    def get_database(self, project_id, instance, database):
+        """
+        Retrieves a database resource from a Cloud SQL instance.
+
+        :param project_id: Project ID of the project that contains the instance.
+        :type project_id: str
+        :param instance: Database instance ID. This does not include the project ID.
+        :type instance: str
+        :param database: Name of the database in the instance.
+        :type database: str
+        :return: A Cloud SQL database resource, as described in
+            https://cloud.google.com/sql/docs/mysql/admin-api/v1beta4/databases#resource
+        :rtype: dict
+        """
+        return self.get_conn().databases().get(
+            project=project_id,
+            instance=instance,
+            database=database
+        ).execute(num_retries=NUM_RETRIES)
+
+    def create_database(self, project, instance, body):
+        """
+        Creates a new database inside a Cloud SQL instance.
+
+        :param project: Project ID of the project that contains the instance.
+        :type project: str
+        :param instance: Database instance ID. This does not include the project ID.
+        :type instance: str
+        :param body: The request body, as described in
+            https://cloud.google.com/sql/docs/mysql/admin-api/v1beta4/databases/insert#request-body
+        :type body: dict
+        :return: True if the operation succeeded, raises an error otherwise
+        :rtype: bool
+        """
+        response = self.get_conn().databases().insert(
+            project=project,
+            instance=instance,
+            body=body
+        ).execute(num_retries=NUM_RETRIES)
+        operation_name = response["name"]
+        return self._wait_for_operation_to_complete(project, operation_name)
+
+    def patch_database(self, project, instance, database, body):
```

Comment:
Hmm.. Lets make this `update_database`. There was something similar in the last PR as well. Can we change patch to update in that too?

Reply:
Actually, in Cloud SQL, both for instances and for databases, [patch](https://cloud.google.com/sql/docs/mysql/admin-api/v1beta4/instances/patch) and [update](https://cloud.google.com/sql/docs/mysql/admin-api/v1beta4/instances/update) are two different operations.

I've decided for now to implement only the patch operator, however in the future there could be a separate update operator, too.

So I'd rather stick to the same names that Cloud SQL uses to avoid any confusion.

Reply:
Ya. Makes sense then.

Reply:
I didn't know that they are 2 different operatons. I am still trying to figure out the difference between both the operations. Looks like patch does partial update only.

Reply:
Yes, sorry, forgot to explain this. You're right, `patch` is a partial update, while `update` is a full update.